plugin.xml

```diff
@@ -45,7 +43,8 @@
         <source-file src="src/android/Notification.java" target-dir="src/com/cowbell/cordova/geofence" />
         <source-file src="src/android/ReceiveTransitionsIntentService.java" target-dir="src/com/cowbell/cordova/geofence" />
         <source-file src="src/android/RemoveGeofenceCommand.java" target-dir="src/com/cowbell/cordova/geofence" />
-
+		    
+		<framework src="com.google.android.gms:play-services:+" />
```

Comment:
We only need `play-services-location:+` here. Also please ensure using spaces not tabs along the project.